### PR TITLE
Solutions

### PR DESCRIPTION
**Pull request by circlecube**
## Proposed changes

This is the main solutions branch. Any plugin work related to solutions should target this branch in a PR. Once all is complete this will merge to main for the solutions release.

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by circlecube** [comment]
This is feature complete and I've combined the `3.15.0` release updates into this branch/PR and updated the PR to merge into `main`. Once tests all pass and code is approved, I'll merge and tag a release for Solutions work!

The display of any solutions components or pages requires either a `hasSolution` capability, which customers only receive once they have purchased a solution.

The only exception to this is the Solutions Banner on the plugin home page, and that requires a `abTestSolutionsLaunched` experiment capability. We can set this per site for testing and once we have the greenlight, we can ramp that up to 100 or whatever percentage rollout we want to use.

**Review by wpscholar — APPROVED** [review]

**Pull request merged**
